poster/poster.tex

```diff
@@ -191,17 +203,7 @@ \section*{BackgroundSub}
 \end{center}
 
 \para
-Curabitur vitae diam non enim vestibulum interdum. Duis ante orci,
```

Comment:
I am paranoid we would accidentally leave some of this in the final version, so removing it now before it is too late! (Also removing now might lessen chances of merge conflicts when both of us work on different sections concurrently because if this is left in, we might find that we need to remove some here and there to keep this at one page length.)

But just so we don't get empty section, I put much shorter English gibberish in its place. 😬